filters/filter_fuzzy.go

```diff
@@ -42,7 +42,8 @@ func (filter *fuzzyFilter) Match(alert *models.UnseeAlert, matches int) bool {
 		}
 
 		if alert.Silenced != "" {
-			if silence, found := store.SilenceStore.Store[alert.Silenced]; found {
+			silence := store.Store.GetSilence(alert.Silenced)
```

Comment:
You could potentially collapse these two lines into one:

```
if silence := store.Store.GetSilence(alert.Silenced); silence != nil {
```

But that's just a stylistic thing I guess. It also keeps the `silence` variable locally-scoped to the `if` block but that probably doesn't matter.